src/Adjudications/AdjudicationHistory/AdjudicationHistoryContainer.test.js

```diff
@@ -37,6 +37,6 @@ describe('Adjudication History container', () => {
       />
     )
 
-    expect(wrapper.dive()).toMatchSnapshot()
+    expect(wrapper.dive().debug()).toMatchSnapshot()
```

Comment:
Not sure about `.debug()`.  Generally would only try to use this when debugging.  I still think we should probably be using react-test-renderer for snapshot stuff.

Reply:
Found one of the enzyme developers providing it as a possible solution here: https://github.com/airbnb/enzyme/issues/1533#issuecomment-367392996

A bit reluctant to do a whole render as it would break encapsulation of the wrapped components - I'd have to put in loads of info into this test about how to set up the descendent's item state correctly (It wraps `Page` which requires quite a lot of state). 

The snapshot would also then contain all the information about how they are rendered - so any changes to them would break this test. This test is attempting to just test the structure of this component - the other components are tested in their own unit tests.

Reply:
It seems like jest-24 requires [custom](https://github.com/facebook/jest/issues/7802#issuecomment-460604154) [serializer](https://github.com/kentcdodds/jest-glamor-react) registering. That would allow moving away from debug